### DISCUSSION TITLE
Usage of dueDateChangedByRecall field in request record

### DISCUSSION THREAD

**Discussion by nassibnassar** [folio-org/folio-analytics#476] [Discussions]
From Brooks Travis:
> How are you all currently using the `dueDateChangedByRecall` field on the request record in your reports (if you are)? We’re trying to come up with a solution for [CIRC-1112](https://issues.folio.org/browse/CIRC-1112) and are concerned that the proposed fix might break things for you all. Any thoughts?

**Comment by amzoss** [comment]
We do have a use case for dueDateChangedByRecall (https://issues.folio.org/browse/UXPROD-2017), but first we need the additional field dueDateChangedByRecallDate as described in:

https://issues.folio.org/browse/UXPROD-2970
https://issues.folio.org/browse/UXPROD-2971

**Reply by amzoss** [reply]
I've also added a long comment to CIRC-1112
